tests/resources.py

```diff
@@ -0,0 +1,13 @@
+from time_tracker_api.database import AuditedModel
+from time_tracker_api.sql_repository import db, SQLAuditedModel
+
+
+class PersonSQLModel(db.Model, SQLAuditedModel):
```

Comment:
Because this is the SQLAlchemy model, not a generic one

Reply:
is it extremelly necessat to include `SQL` as part of the name?

Reply:
Yes, because this model is specific to the SQL implementation. As well as I have the SQL prefix in the repository and other resources related to this particular technology, which has its own characteristics: specific library, handling of commits, rollbacks, the concept of tables, columns and so forth, which is no the same as, for instance, an In-memory implementation, a Mongo Implementation, etc.

Reply:
Just by looking it has the SQL prefix you know that this has SQL-specific logic.

Reply:
We are not going to add support for Mongo, in-memory or so. We should not pollute code with implementation details.